<ui version="4.0" >
 <class>LabelTab</class>
 <widget class="QWidget" name="LabelTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="_Label_4" >
     <property name="text" >
      <string>Default plot font &amp;size:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>_globalLabelFontSize</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QDoubleSpinBox" name="_globalLabelFontSize" >
     <property name="decimals" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QPushButton" name="_globalLabelBold" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="shortcut" >
      <string>Alt+B</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QPushButton" name="_globalLabelItalic" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="shortcut" >
      <string>Alt+I</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="_Label_7" >
     <property name="text" >
      <string>Default font famil&amp;y:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>_globalLabelFontFamily</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6" >
    <widget class="QFontComboBox" name="_globalLabelFontFamily" />
   </item>
   <item row="2" column="0" colspan="7" >
    <widget class="Line" name="line_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="TextLabel1_2_2_2" >
     <property name="text" >
      <string>Insert scalar in label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>_scalars</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="5" >
    <widget class="Kst::ScalarSelector" native="1" name="_scalars" />
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="TextLabel1_2_2_3" >
     <property name="text" >
      <string>Insert string in label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>_scalars</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="5" >
    <widget class="Kst::StringSelector" native="1" name="_strings" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="_Label_13" >
     <property name="text" >
      <string>&amp;Top label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>_topLabelText</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="6" >
    <widget class="Kst::LabelLineEdit" name="_topLabelText" />
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="_Label_14" >
     <property name="text" >
      <string>Botto&amp;m label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>_bottomLabelText</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="6" >
    <widget class="Kst::LabelLineEdit" name="_bottomLabelText" />
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="_Label_15" >
     <property name="text" >
      <string>&amp;Left label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>_leftLabelText</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="6" >
    <widget class="Kst::LabelLineEdit" name="_leftLabelText" />
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="_Label_16" >
     <property name="text" >
      <string>&amp;Right label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>_rightLabelText</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="6" >
    <widget class="Kst::LabelLineEdit" name="_rightLabelText" />
   </item>
   <item row="9" column="0" colspan="7" >
    <widget class="Line" name="line" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="10" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="_showLegend" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Show le&amp;gend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_editLegendContents" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>E&amp;dit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="3" colspan="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>376</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6" >
    <widget class="Kst::ColorButton" name="_globalLabelColor" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ScalarSelector</class>
   <extends>QWidget</extends>
   <header>scalarselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::StringSelector</class>
   <extends>QWidget</extends>
   <header>stringselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::LabelLineEdit</class>
   <extends>QLineEdit</extends>
   <header>labellineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_topLabelText</tabstop>
  <tabstop>_bottomLabelText</tabstop>
  <tabstop>_leftLabelText</tabstop>
  <tabstop>_rightLabelText</tabstop>
  <tabstop>_globalLabelFontSize</tabstop>
  <tabstop>_globalLabelBold</tabstop>
  <tabstop>_globalLabelItalic</tabstop>
  <tabstop>_globalLabelColor</tabstop>
  <tabstop>_globalLabelFontFamily</tabstop>
  <tabstop>_showLegend</tabstop>
  <tabstop>_editLegendContents</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
